resources/sass/_base.scss

```diff
@@ -379,25 +380,27 @@ $caret-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5
   }
 
   dt:hover + dd > ul > li.kint-active-tab {
-    border-color: $border-color-hover;
-    color: $variable-type-color-hover;
+    border-color: var(--border-color-hover);
+    color: var(--variable-type-color-hover);
   }
 
   dt > .kint-color-preview {
-    width: $color_size * 1px;
-    height: $color_size * 1px;
+    width: var(--color-size);
+    height: var(--color-size);
     display: inline-block;
     vertical-align: middle;
     margin-left: 10px;
-    border: $border;
+    border: var(--border);
+    max-width: 30px;
+    max-height: 30px;
     background: {
       color: #ccc;
       image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2 2"><path fill="%23FFF" d="M0 0h1v2h1V1H0z"/></svg>');
-      size: if($color_size >= 30, 30px, 100%);
+      size: 100%;
```

Comment:
I'm not sure these changes won't break some layouts but I'll merge them first and test them myself later

Reply:
I don't see why the new approach would break anything, but we could also go this route as well:
`size: min(30px, 100%)`
which has the same effect.

Alternatively, if you found a breaking change with either of these CSS-only implementations, we can revert back to the SCSS solution.